packages/banner/src/__stories__/Banner.stories.js

```diff
@@ -4,7 +4,8 @@ import { storiesOf } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
```

Comment:
Each component package will contain its own stories. However, storybook will be run for all components from the project root.